deploy/playbooks/03_app.yml

```diff
@@ -3,26 +3,33 @@
 
   tasks:
     - name: Clone the repository to a specific version (to a temp location)
-      git:
+      ansible.builtin.git:
         repo: "{{ repository_url }}"
         dest: /tmp/src
-        accept_hostkey: yes
+        accept_hostkey: true
         version: "{{ app_version }}"
 
     - name: Build with a given commit hash
       # This will be stored in local registry, and available as version to docker-compose
       # where we can just reference correct version
-      shell: "cd /tmp/src && make docker/build V={{ app_version }}"
+      ansible.builtin.command:
+        chdir: /tmp/src
+        cmd: "/usr/bin/make docker/build V={{ app_version }}"
+      register: output
+      changed_when: output.rc != 0
+      failed_when: output.rc != 0
```

Comment:
This does not make sense. How should ansible know if this has failed or changed if the conditional is the same?
For ansible:
changed = continue
failed = stop here

Reply:
this is a stupid hack for the linter, because otherwise the linter complains that it can't predict if the thing is going to be idempotent or not. Or something like that. Point is – it's not useful for humans, it's just to make the linter happy.